hazelcast-jet-python/src/main/java/com/hazelcast/jet/python/PythonServiceConfig.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.python;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.io.File;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Configuration object for the Python service factory.
+ */
+public class PythonServiceConfig implements Serializable {
+    private File baseDir;
+    private File handlerFile;
+    private String handlerModule;
+    private String handlerFunction;
```

Comment:
I wonder if it's better to represent these internally as `Properties` class or similar, so we don't have to add a method for every single option.

Reply:
There is also important Javadoc attached to each setter, and this way the available options are easily discoverable.

Reply:
You mean a field? Methods should stay as they have important Javadoc on them and make all the options easily discoverable.